package.json

```diff
@@ -11,8 +11,8 @@
   "scripts": {
     "test": "jest app/javascript/ --coverage",
     "test:watch": "jest app/javascript/ --watch",
-    "storybook": "start-storybook -p 6006 -c app/javascript/.storybook -s app/javascript/.storybook/assets",
-    "build-storybook": "build-storybook -c app/javascript/.storybook -s app/javascript/.storybook/assets",
+    "storybook": "start-storybook -p 6006 -c app/javascript/.storybook -s app/assets",
```

Comment:
Storybook uses the app's actual assets folder now for static assets. This makes more sense than copying over what is needed.